### PR TITLE
Add columnValueMap option to select PhpExcel value method

### PR DESCRIPTION
**Pull request by ozee31**

### CODE REVIEW

File: src/Excel/Manager.php [hunk 1]
```diff
@@ -113,6 +114,19 @@ public function read(PHPExcel_Worksheet $worksheet, Table $table, array $options
         return $entities;
     }
 
+    private function getCellValue($cell, $column, $options)
```
Comment (robertpustulka) [inline src/Excel/Manager.php]: I'm following the CakePHP convention with private/protected methods, so this should be `protected`.

**Review by robertpustulka — CHANGES_REQUESTED** [review]

File: src/Excel/Manager.php [hunk 2]
```diff
@@ -113,6 +114,19 @@ public function read(PHPExcel_Worksheet $worksheet, Table $table, array $options
...
+    {
+        $valueMethod = (isset($options['columnValueMap'][$column])) ? $options['columnValueMap'][$column] : 'formated';
+
+        switch ($valueMethod) {
+            case 'formated': return $cell->getFormattedValue();
```
Comment (robertpustulka) [inline src/Excel/Manager.php]: A typo here. Should be `formatted`?

**Comment by robertpustulka** [comment]
@ozee31 are you still interested in that change?

**Comment by ozee31** [comment]
I applied the requested changes

**Comment by robertpustulka** [comment]
@ozee31 You need to update tests and docs as well. Test are broken now.

**Comment by robertpustulka** [comment]
@ozee31 you still have some build errors:
https://travis-ci.org/robotusers/cakephp-excel/builds/265536376

**Pull request closed without merging**